src/ray/raylet/node_manager.h

```diff
@@ -237,6 +237,11 @@ class NodeManager : public rpc::NodeManagerServiceHandler,
       int64_t limit,
       const std::function<void()> &on_all_replied);
 
+  /// Get the information of the agent process when the agent finished register.
+  ///
+  /// \return The information of the agent process.
+  inline rpc::AgentInfo SyncGetAgentInfo() { return agent_manager_->SyncGetAgentInfo(); }
```

Comment:
should it be const? Also remove inline which is not necessary here.

Reply:
done.